Dfe.Academies.External.Web/Pages/School/ApplicationSelectSchool.cshtml.cs

```diff
@@ -63,7 +63,7 @@ public ApplicationSelectSchoolModel(ILogger<ApplicationSelectSchoolModel> logger
 		    _conversionApplicationCreationService = conversionApplicationCreationService;
 		}
 
-		public async Task OnGetAsync()
+		public void OnGetAsync()
```

Comment:
Why is this not Async?

Reply:
And shouldn't it be an override?

Reply:
it just gets data out of cache now. Now not calling API so no need to be async. No need to be an override either. dot net looks for Onget() / ongetasync() / onpost() / onpostasync() . None of the other methods are overrides

Reply:
I know you've merged this now but could this be renamed to `OnGet()` because it's not async anymore?